allennlp/commands/train.py

```diff
@@ -104,6 +110,14 @@ def train_model_from_args(args: argparse.Namespace):
     # Import any additional modules needed (to register custom classes)
     for package_name in args.include_package:
         import_submodules(package_name)
+
+    if not args.cont and os.path.exists(args.serialization_dir):
+        raise Exception(f"Serialization directory ({args.serialization_dir}) already exists.  "
```

Comment:
Should I instead `logger.error` and then `sys.exit(1)`?  The output would be more friendly.